pkg/coinmarketcap/coinmarketcap.go

```diff
@@ -10,12 +10,15 @@ import (
 )
 
 const (
-	// Bitcoin is the constant for the Bitcoin currency
+	// Constant for the Bitcoin currency
```

Comment:
This doesn't play well with the godoc tool. We should rather have first the name of the constant and then what it means.